### PR TITLE
[FLINK-7494][travis] Add license headers to '.travis.yml' file

### PR DESCRIPTION
**Pull request by yew1eb**
Just add license headers to '.travis.yml' file.
:smile:

### CODE REVIEW

**Comment by yew1eb** [comment]
Hi @twalthr  @zentol,
Please helps to review, Thank you very much. :beers:

**Comment by yew1eb** [comment]
CC @aljoscha :satisfied:

**Comment by yew1eb** [comment]
CC @zentol

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,19 @@
-# s3 deployment based on http://about.travis-ci.org/blog/2012-12-18-travis-artifacts/
```
Comment (zentol) [inline .travis.yml]: this line shouldn't be removed

Comment (yew1eb) [reply .travis.yml]: done.

**Pull request closed without merging**